modules/tensor_mechanics/src/materials/ADMultiplePowerLawCreepStressUpdate.C

```diff
@@ -0,0 +1,144 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "ADMultiplePowerLawCreepStressUpdate.h"
+
+registerMooseObject("TensorMechanicsApp", ADMultiplePowerLawCreepStressUpdate);
+
+InputParameters
+ADMultiplePowerLawCreepStressUpdate::validParams()
+{
+  InputParameters params = ADRadialReturnCreepStressUpdateBase::validParams();
+  params.addClassDescription(
+      "This class uses the stress update material in a radial return isotropic power law creep "
+      "model. This class can be used in conjunction with other creep and plasticity materials "
+      "for more complex simulations.");
+
+  // Linear strain hardening parameters
+  params.addCoupledVar("temperature", "Coupled temperature");
+  params.addRequiredParam<std::vector<Real>>("coefficient",
+                                             "Leading coefficient in power-law equation");
+  params.addRequiredParam<std::vector<Real>>("n_exponent",
+                                             "Exponent on effective stress in power-law equation");
+  params.addRequiredParam<std::vector<Real>>("m_exponent",
+                                             "Exponent on time in power-law equation");
+  params.addRequiredParam<std::vector<Real>>("stress_thresholds",
+                                             "Stress intervals to switch creep behavior");
+  params.addRequiredParam<std::vector<Real>>("activation_energy", "Activation energy");
+  params.addParam<Real>("gas_constant", 8.3143, "Universal gas constant");
```

Comment:
Values for _R_ found in the MOOSE code base

- 8.314
- 8.3143
- 8.31434
- 8.314472
- 8.3144598

👀

Reply:
Any preference?